packages/eas-json/src/EasJsonSchema.ts

```diff
@@ -13,7 +13,8 @@ const AndroidGenericSchema = Joi.object({
 const AndroidManagedSchema = Joi.object({
   workflow: Joi.string().valid('managed').required(),
   credentialsSource: Joi.string().valid('local', 'remote', 'auto').default('auto'),
-  buildType: Joi.string().valid('apk', 'app-bundle').default('app-bundle'),
+  releaseChannel: Joi.string(),
+  buildType: Joi.string().valid('apk', 'app-bundle', 'development-client').default('app-bundle'),
```

Comment:
Why did you already add `development-client` here? I think we still don't support it.

Reply:
it's supported in turtle-v2, but incorrectly for now(`:app:assembleRelease`). I won't include that in docs, so I think it it's ok to leave it. I'm using types from `@expo/eas-build-job` so `development-client` is already included there in the type